components.json

```diff
@@ -816,6 +816,7 @@
 		},
 		"sas": {
 			"title": "SAS",
+			"require": ["sql", "lua", "groovy"],
```

Comment:
I know that you plan to add support for Lua and Groovy but maybe this should be its own PR. I really want to merge this soon.

Reply:
Ok, i'll cover lua and groovy in the next pull request after we merge this one.

Reply:
Nice. Please remove the unnecessary requires in the meantime.

Also, instead of `require`, maybe we could use `peerDependencies` instead. This will make SQL and the others optional so that users can make smaller Prism builds and aren't forced to include SQL (and others) which they might not even use. 
You don't even need to change your language definition fo this. The only real change will be for tests, where you will have to [manually include SQL](https://prismjs.com/test-suite.html#writing-tests) and the test page where you have to manually load SQL before SAS (simply click on SQL before clicking on SAS).

Reply:
The commit I just made removes the other requirements. I didn't swap to peerDependencies. But I could if you really wanted. PROC SQL is part of base sas, so it's fairly commonly used.

Reply:
If it's common, then we'll leave it as `require`.